packages/module/src/index.ts

```diff
@@ -57,6 +57,9 @@ export * from './MessageBar';
 export { default as MessageBox } from './MessageBox';
 export * from './MessageBox';
 
+export { default as Popover } from './Popover';
```

Comment:
The updates to this file shouldn't be necessary? The Popover component was renamed to ChatbotPopover.

Try cleaning and building again and see if this change goes away

Reply:
This is all set!